src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -66,12 +71,49 @@ def setUp(self):
         self.dm_cmd.get_params(self)
         self.processes = self.params.get("np", '/run/datamover/processes/*')
 
-    # The valid parameter types for setting param locations.
-    PARAM_TYPES = ("POSIX", "DAOS_UUID", "DAOS_UNS")
+        # List of test paths to create and remove
+        self.posix_test_paths = []
+
+    def pre_tear_down(self):
+        """Tear down steps to run before tearDown().
+
+        Returns:
+            list: a list of error strings to report at the end of tearDown().
+
+        """
+        error_list = []
+        # Remove the created directories
+        if self.posix_test_paths:
+            command = "rm -rf {}".format(self.get_posix_test_path_string())
+            try:
+                self._execute_command(command)
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no '_execute_command' member